generator/src/generator.ts

```diff
@@ -1669,6 +1669,47 @@ export class Generator {
     if (this.combineSubpaths) map.combineSubpaths();
     return map.toJSON();
   }
+
+  /**
+   *
+   * @param outputDirectory The output directory, resolved relative to process.cwd(), where the import
+   * map and its files are stored on the file system. Defaults to 'dist'
+   * @param importMapFileName The name of the import map file stored to disk, within the outputDirectory. Defaults to
+   * main.importmap
+   * @param hostedBaseUrl a fully qualified or relative string path used as base url for all URLs within the import
+   * If not provided, hostedBaseUrl defaults to './'
+   */
+  async downloadAll(
+    outputDirectory: string = 'deps',
+    importMapFileName: string = 'main.importmap',
+    hostedBaseUrl: string = './'
+  ) {
+    if (!isNode) {
+      throw new JspmError('downloadAll is only available in nodejs');
+    }
+
+    const fs = await import('node:fs/promises');
+    const path = await import('node:path');
```

Comment:
We need to ensure these don't interfere with the browser build, I've been wrapping them in `eval()` to do that, there's likely a better way but worth noting.

Reply:
Is the browser build done by user code that bundles the published code? Running `chomp build` without the `eval()` results in a runtime `import()` that would be ignored if executed as-is within the browser. However, webpack/other bundlers would process if jspm generator is bundled by user code.

I'll change it to `eval` like you suggested, to be consistent. As an alternative, would moving the module names into variables avoid them being bundled in the major bundlers? Not sure how sophisticated they are nowadays.

```js
const modules = ['node:fs/promises', 'node:'path'];
const [fs, path] = await import(modules);
```

Reply:
The variables approach is also a good one, although it can become a game of cat and mouse with static analysis.

These days the better approach for Node.js builtins is actually to just use `process.getBuiltinModule`. I believe it might work for our Node.js compat these days. Can we use that here instead perhaps? Would be nice to replace the other usages of Node.js builtins with this approach too.